.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -2,24 +2,37 @@
 name: Bug Report
 about: Create a report to help us improve
 title: ''
-labels: Support
+labels: "Support, needs-triage"
 
 ---
 
 <!-- Please use this template while reporting a bug and provide as much info as possible. Thanks!-->
 <!-- 请使用英文描述问题 -->
+<!-- Note, lacking of information will delay the handling of issue
+     See our github issue handling flow here:
+     https://github.com/emqx/emqx/blob/master/.github/ISSUE_TEMPLATE/asserts/issue-handling.png
+-->
 
-**Environment**:
+### Subject of the issue
+Describe your issue here.
 
+### Your environment
 - EMQ X version (e.g. `emqx_ctl status`): 
 - Hardware configuration (e.g. `lscpu`):
 - OS (e.g. `cat /etc/os-release`):
 - Kernel (e.g. `uname -a`):
 - Erlang/OTP version (in case you build emqx from source code): 
 - Others:
 
-**What happened and what you expected to happen**:
+### Steps to reproduce
+Tell us how to reproduce this issue. 
 
-**How to reproduce it (as minimally and precisely as possible)**:
+It would be great if you can attach the log archive generated by [node_dump tool](https://github.com/emqx/emqx/blob/master/bin/node_dump) 
 
-**Anything else we need to know?**:
+### Expected behaviour
+Tell us what should happen
```

Comment:
```suggestion
Please describe the expected behaviour
```

Reply:
I will keep the original one since it `Please describe the expected behaviour` feels the same as the title "Expected behaviour"

btw, I borrow it from 
https://github.com/angular-translate/angular-translate/blob/master/.github/ISSUE_TEMPLATE.md